flake8_html/plugin.py

```diff
@@ -179,6 +198,8 @@ def finished(self, filename):
         rendered = self.source_template.render(**params)
         with codecs.open(source_filename, 'w', encoding='utf8') as f:
             f.write(rendered)
+        # stop() isn't called when, f.ex. called via some CI runners, so call it
+        self.stop()
```

Comment:
I have a feeling that those circumstances can be avoided if I just write my own flake8 reporter for the test runner I'm using for Django. Putting that self.stop() there has seemed like a 💩-mix since I put it there, and I'm not a firm believer in 💩. I'm removing the call to self.stop().